app/src/main/java/com/kks/nimblesurveyjetpackcompose/model/response/MetaResponse.kt

```diff
@@ -16,3 +17,11 @@ data class MetaResponse(
     @Json(name = "message")
     val message: String?
 )
+
+fun MetaResponse.toMeta() = Meta(
+    page = page ?: 0,
+    pages = pages ?: 0,
+    pageSize = pageSize ?: 0,
+    records = records ?: 0,
+    message = message.orEmpty()
+)
```

Comment:
If we always want a default value of `Meta`, we can update this to
```suggestion
fun MetaResponse?.toMeta() = Meta(
    page = this?.page ?: 0,
    pages = this?.pages ?: 0,
    pageSize = this?.pageSize ?: 0,
    records = this?.records ?: 0,
    message = message.orEmpty()
)
```